Core/CodeGeneration/ReflectionEmit/ReflectionEmitCodeGenerator.cs

```diff
@@ -78,7 +78,11 @@ private class ModuleContext
 
     public DebugInfoGenerator DebugInfoGenerator
     {
+#if FEATURE_PDBEMIT
```

Comment:
Please rewrite to only toggle the actual selection of the instance but don't duplicate the logic. In cases such as this, a larger code (format) change with only a small feautre-toggle is preferred to keep behavorial changes to a minumum.

Reply:
I can work with the new code, but I was thinking in the line of this:
```
if ( _moduleContext.DebugInfoGenerator == null)
{
#if FEATURE_PDBEMIT
  var generator = DebugInfoGenerator.CreatePdbGenerator();
#else
  var generator = NullDebugInfoGenerator.Instance;
#endif
  _moduleContext.DebugInfoGenerator = generator;
}
return _moduleContext.DebugInfoGenerator;